code/modules/cmc/crew.dm

```diff
@@ -336,16 +336,15 @@ HOLOMAP PROCS
 		user.client.screen -= holomap_tooltips[uid]
 
 	if(holomap_images[uid])
-		holomap_images[uid].len = 0
```

Comment:
why?

Reply:
found a runtime [18:09:50] Runtime in crew.dm,387: Cannot modify null.len.
couldnt find the exact reason why it was null at the time, but figured making it a new empty list would give the same effect as clearing the list. i did it with the other .len = 0 instances in the file too for ease of reading. can be reverted if ye want

Reply:
It's checked on line 338 for a non-null value, something else must be going on.

Reply:
Nevermind, got the line numbers mixed up. This isn't necessarily horrible but you should figure out if `holomap_tooltips[uid]` being `null` at line 386 is actually something that's not supposed to happen

Reply:
it's set to null on deactivate, i mostly changed that line for consistency. it has the same effect, at least.

Reply:
@PaulRitter you coded this, what the fuck is going on

Reply:
wait what the fuck. how the hell is this runtiming, shouldn't if(holomap_images[uid]) prevent exactly this?

Reply:
this genuinly should never happen cause before this gets called, `initializeUser` should set it. `deactive` clears it and disconnects the user from the holomap

Reply:
but yeah the fix is fine

Reply:
initializeuser is only called when you click the pc, not when you're already in it.
you can still have the interface of the computer open even when the /deactivate function sets shit to null
at least thats my explanation